GenderPayGap.WebUI/Views/ChangePersonalDetails/ChangePersonalDetails.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.Account.ChangePersonalDetailsViewModel
+
+@{
+    ViewBag.Title = "Change your personal details - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Change your personal details
+        </h1>
+            
+        <form method="POST" action="@Url.Action("ChangePersonalDetailsPost", "ChangePersonalDetails")">
+            @(Html.AntiForgeryToken())
+            
+            @(Html.GovUkErrorSummary())

```

Comment:
I think this `@(Html.GovUkErrorSummary())` should go above the `<h1>` (although we're not always consistent about this)